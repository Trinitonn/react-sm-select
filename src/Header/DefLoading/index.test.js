import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { DefLoading } from './index';

describe('DefLoading', () => {
  it('should render properly', () => {
    const wrapper = shallow(<DefLoading />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
