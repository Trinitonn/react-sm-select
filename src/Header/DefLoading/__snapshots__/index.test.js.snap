// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefLoading should render properly 1`] = `
<span
  className="Header__loading"
>
  <span
    className="Header__loading__icon"
  />
</span>
`;
